Harvest.Web/ClientApp/src/Quotes/QuoteContainer.tsx

```diff
@@ -179,7 +179,7 @@ export const QuoteContainer = () => {
       body: JSON.stringify(quote),
     });
 
-    setNotification(request, "Saving Quote", "Quote Saved");
+    setNotification(request, "Submitting Quote", "Quote Saved");
```

Comment:
You need to check if it is submitting or saving.
Then change the "Saved" part as well